ios/RNSScreenStack.mm

```diff
@@ -508,45 +593,22 @@ - (void)updateContainer
 #endif
 }
 
-// By default, the header buttons that are not inside the native hit area
-// cannot be clicked, so we check it by ourselves
-- (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
```

Comment:
We should bring this method back on Fabric and check a test concerning it to see if it works correctly.

Reply:
Noted down. Leaving it for new PR

Reply:
f61d67c